src/site/xdoc/checks/coding/missingswitchdefault.xml

```diff
@@ -140,8 +140,6 @@ class Example3 {
       case GREEN:
         System.out.println("GREEN");
         break;
-      default:
-        System.out.println("Something else");
```

Comment:
You should do this removal from example java file.
Please find this code in java file.
After you run `mvn clean test` it will update ths xml by such removal.